adyenv6core/src/com/adyen/v6/service/AdyenPaymentService.java

```diff
@@ -74,8 +73,9 @@
     List<PaymentMethod> getPaymentMethods(BigDecimal amount, String currency, String countryCode, String shopperLocale, String shopperReference) throws IOException, ApiException;
 
     /**
-     * @deprecated use getPaymentMethods including shopperLocale instead
-     * {@link #getPaymentMethods(BigDecimal amount, String currency, String countryCode, String shopperLocale)
+     * Get Payment methods using HPP Directory Lookup
+     *
+     * @deprecated use getPaymentMethods including shopperReference instead {@link #getPaymentMethods(BigDecimal amount, String currency, String countryCode, String shopperLocale)
```

Comment:
Please use the method including _String shopperReference_ in the link

Reply:
done